src/SourceBuild/content/eng/build.sourcebuild.targets

```diff
@@ -111,21 +83,22 @@
           Inputs="$(MSBuildProjectFullPath)"
           Outputs="$(BaseIntermediateOutputPath)ReportPoisonUsage.complete" >
     <ItemGroup>
-      <FinalCliTarball Include="$(SharedOutputPath)**/*$(ArchiveExtension)" />
+      <!-- Exclude the Private.SourceBuilt.Artifacts archive from poison usage scan. -->
```

Comment:
Would it make sense to move this comment to be directly above the line that is doing the removal?